### PR TITLE
rivet: 2.7.0 -> 2.7.1

### PR DESCRIPTION
**Pull request by veprbl**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by srhb** [comment]
Retriggering eval due to master failure
@GrahamcOfBorg eval

**Comment by xeji** [comment]
x64_64-linux build fails in the sandbox (but succeeds with sandboxing turned off).

File: pkgs/development/libraries/physics/rivet/default.nix [hunk 1]
```diff
@@ -28,6 +28,8 @@ stdenv.mkDerivation rec {
   propagatedBuildInputs = [ fastjet ghostscript gsl yoda ];
 
   preConfigure = ''
+    substituteInPlace analyses/Makefile.in \
+      --replace "!(tmp)" ""
```
Comment (timokau) [inline pkgs/development/libraries/physics/rivet/default.nix]: What does this achieve?

**Comment by timokau** [comment]
The failure looks like you need to run `patchShebangs`.

File: pkgs/development/libraries/physics/rivet/default.nix (hunk 1 above)
Comment (veprbl) [reply pkgs/development/libraries/physics/rivet/default.nix]: I don't know. This "!(tmp)" piece was inserted by upstream into the tarball release and it is not present in the official repository, it also breaks the build.

**Comment by veprbl** [comment]
Thanks... #45141

**Pull request merged**